src/safeposix/syscalls/net_calls.rs

```diff
@@ -3252,12 +3411,14 @@ impl Cage {
                     }
                     SOL_SOCKET => {
                         // Here we check and set socket_options
+                        // we store this flag in socket_options at the corresponding bit
                         let optbit = 1 << optname;
                         let sock_tmp = sockfdobj.handle.clone();
                         let mut sockhandle = sock_tmp.write();
 
                         match optname {
                             SO_ACCEPTCONN | SO_TYPE | SO_SNDLOWAT | SO_RCVLOWAT => {
```

Comment:
A little explanation about these flags would be great for understanding, if not already explained anywhere else.